lib/core/importer.ts

```diff
@@ -20,35 +28,49 @@ export class Importer {
     return Importer.instance;
   }
 
-  static async load(patterns: string[]): Promise<AutoClasses> {
+  static load(patterns: string[]): AutoClasses {
     const importer = Importer.getInstance();
-    const pathNames = await importer.matchGlob(patterns);
-    const foundClasses = await Promise.all(pathNames.map((pathName) => importer.scan(pathName)));
+    const pathNames = importer.matchGlob(patterns);
+    const foundClasses = pathNames.map((pathName) => importer.scan(pathName));
     return foundClasses.reduce(
       (merged, found) => ({
         providers: [...merged.providers, ...found.providers],
         controllers: [...merged.controllers, ...found.controllers],
+        exports: [...merged.exports, ...found.providers],
       }),
-      { providers: [], controllers: [] } as AutoClasses,
+      { providers: [], controllers: [], exports: [] },
```

Comment:
we don't need this do we?

```suggestion
exports: [...merged.exports, ...found.providers],
```

Reply:
Thank you. Fixed! 😆